bin/nrnivmodl_core_makefile.in

```diff
@@ -46,7 +46,6 @@ INCLUDES = -I$(CORENRN_INC_DIR)/coreneuron/utils/randoms $(INCFLAGS) -I$(CORENRN
 ifeq (@CORENRN_ENABLE_MPI_DYNAMIC@, OFF)
   INCLUDES += $(if @MPI_CXX_INCLUDE_PATH@, -I$(subst ;, -I,@MPI_CXX_INCLUDE_PATH@),)
 endif
-INCLUDES += $(if @reportinglib_INCLUDE_DIR@, -I$(subst ;, -I,@reportinglib_INCLUDE_DIR@),)
```

Comment:
There is no sonata support in this file, what do?

Reply:
This is Ok. Sonata header include is not required. (I believe this is passed to `nrnivmodl-core` directly)